new_lineage_panel/src/TableDetails.tsx

```diff
@@ -297,12 +299,26 @@ const TableDetails = () => {
         setConfidence,
         setMoreTables,
         setCollectColumns,
-        flow
+        flow,
+        getSessionId()
       );
-    await Promise.all([_bfsTraversal(true), _bfsTraversal(false)]);
+    try {
+      const result = await Promise.all([
+        _bfsTraversal(true),
+        _bfsTraversal(false),
+      ]);
+      if (result.every((isLineage) => !isLineage)) {
+        showNoLineage(_column);
+      }
+    } catch (e) {
+      console.error(e);
+      setSelectedColumn({ table: "", name: "" });
+    }
     endProgressBar();
   };
   if (isLoading || !data || !selectedTable) return <ComponentLoader />;
+  const tabs = ["Column"];
```

Comment:
`useMemo` could be used here for performance improvement. Otherwise, this tabs will be recalculated everytime state changes in this component

Reply:
Should be fine as there is no heavy calculation being performed here. And this portion will probably change in the future.